packages/pangea-node-sdk/tsconfig.json

```diff
@@ -19,6 +19,9 @@
     "noUncheckedIndexedAccess": true,
     "noEmitOnError": true,
     "useDefineForClassFields": true,
+    "target": "es5",
+    "lib": ["es2015", "dom"],
```

Comment:
We shouldn't specify `"dom"` since this is a Node.js library.

Reply:
It should be usable in both, don't know why we have a vanilla-js library when typescript is more common now on the frontend.

Reply:
Removed